core/src/main/java/com/linecorp/armeria/internal/common/TimeoutController.java

```diff
@@ -26,36 +26,36 @@
 public interface TimeoutController {
 
     /**
-     * Schedules a new timeout with the specified {@code timeoutMillis}.
+     * Schedules a new timeout with the specified {@code timeoutNanos}.
      * If a timeout is scheduled already, this method will not start a new timeout.
      *
-     * @param timeoutMillis a positive time amount value in milliseconds.
+     * @param timeoutNanos a positive time amount value in nanoseconds.
      * @return {@code true} if the timeout is scheduled.
      *         {@code false} if the timeout has been scheduled, triggered already
      *         or a timeout cannot be scheduled, e.g. request or response has been handled already.
      */
-    boolean scheduleTimeout(long timeoutMillis);
+    boolean scheduleTimeout(long timeoutNanos);
```

Comment:
How about adding `Nanos` suffix to all methods that accept a `long` duration?

Reply:
+1 That makes the names clearer